slides/05/05.md

```diff
@@ -285,7 +285,7 @@ The $F_1$ score can be generalized to $F_β$ score, where recall is $β$ times
 more important than precision; $F_2$ favoring recall and $F_{0.5}$ favoring
 precision are commonly used.
 
-$$ F_β = \frac{(1 + β)^2 ⋅ \mathrm{precision} ⋅ \mathrm{recall}}{β^2 ⋅ \mathrm{precision} + \mathrm{recall}}
+$$ F_β = \frac{(1 + β^2) ⋅ \mathrm{precision} ⋅ \mathrm{recall}}{β^2 ⋅ \mathrm{precision} + \mathrm{recall}}
        = \frac{\hphantom{()}β^2\mathit{TP~~~~~}+\mathit{~~~~~TP}}{β^2(\mathit{TP}+\mathit{FP})+\mathit{TP}+\mathit{FN}}.$$
```

Comment:
I think on wikipedia the denominator is different.. maybe switch FP and FN?

Reply:
Oh, true. I even have the correct form on the paper, where I counted it, but then I put β^2 to an incorrect place. Will fix.